pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -442,9 +442,14 @@
   aria_label_call_to_action_teaching_the_design_process: "Start the Teaching The Design Process professional learning program"
   aria_label_call_to_action_teaching_csc: "Start the Teaching Computer Science Connections professional learning program"
   aria_label_call_to_action_teaching_ai: "Start the Teaching AI and Machine Learning professional learning program"
+  aria_label_call_to_action_teaching_how_ai_make_decisions: "Start the Teaching How AI Makes Decisions professional learning program"
```

Comment:
@bethanyaconnor I should've told you this yesterday after Turner and I talked, but can you re-work these strings to be in a namespace similar to [this PR](https://github.com/code-dot-org/code-dot-org/pull/59313/files) instead of having them spread out? (Sorry this is annoying and will require so much refactoring 😭)

This will make it easier for Jorge and folks to translate the strings for the launch.

Reply:
hmm, I'm not sure if I can get that change prepped and merged by EOD today. Do you know how strict the today cutoff is? (I can ask in Slack as well)

Reply:
Is this one of the high priority translation pages? If yes, let's keep it as is.

Reply:
It's marked "Yes" on the spec, so I assume so? Not sure how high priority it is, given that I don't think much of self-paced PL is translateable.

Reply:
That page itself looks fully translated (at least in Spanish) so it's probs high priority bc of that. Let's keep this as-is and just give Jorge a heads up that the strings aren't in a namespace.

Reply:
Okay sounds good. Sorry I missed this 😭

Reply:
It's ok!! I should've posted about this before everyone got started.